compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -189,15 +193,6 @@
 
     private Map<BLangBlockStmt, SymbolEnv> blockStmtEnvMap = new HashMap<>();
 
-    public static SemanticAnalyzer getInstance(CompilerContext context) {
```

Comment:
Why did you change the order of the getInstance method? Please revert this change?

Reply:
Sorry, this happened when resolving a merge conflict. Revered now.